src/helpers.ts

```diff
@@ -1,5 +1,8 @@
 /* eslint-disable-next-line */
-import _ from 'lodash'
+import _ from 'lodash'
+import { PolEnum, DirEnum, IXYOperator } from './interfaces'
+import Dimension from './Dimension'
+import Operator from './Operator'
```

Comment:
I strongly prefer to make helpers last resort, not default go-to.

Special operators for `Dimension` and `Operators` should go there, not here.

E.g. clearly `vectorIndicesForParticle` belongs to `Photons`, not here.